.github/workflows/pom_update.yml

```diff
@@ -24,6 +28,10 @@ jobs:
         git checkout -b $BRANCH
         ./mvnw build-helper:parse-version versions:set -DnewVersion=\${parsedVersion.majorVersion}.\${parsedVersion.minorVersion}.\${parsedVersion.nextIncrementalVersion}-SNAPSHOT
         git add pom.xml
+        VERS=$(./mvnw help:evaluate -Dexpression=project.version -q -DforceStdout|tr -d SNAPSHOT|tr -d "-")
+        # Replace version in README.md look for >, :, ` (greater than, colon, backtick) followed by 2 period number period number and replace it with the leading char and new version
+        sed -i -E 's/([\>`\:]{1})2\.[0-9]\.[0-9]/\1'"$VERS"'/g' README.md
```

Comment:
I'm happy to hear suggestions on a better way to handle this. There are other version identifiers in the README but the three we currently care about fulfill this specific prefix condition.